### ISSUE TITLE
Cannot configure openzeppelin-test-helpers a second time

### ISSUE THREAD

**Issue by phmatray** [OpenZeppelin/openzeppelin-test-helpers#69]
With the command `truffle migrate --reset --network kovan`, I have an error raised by my truffle migration script.

It says :
`Cannot configure openzeppelin-test-helpers a second time`

Is this a bug or a misunderstanding on my part?

Here is the faulty code.
```javascript
const Migrations = artifacts.require('Migrations')

require('openzeppelin-test-helpers/configure')({web3})

module.exports = function(deployer) {
  deployer.deploy(Migrations)
}
```

**Comment by phmatray** [comment]
I've found a reference about this problem.
https://github.com/GeoDB-Limited/geodb-federation-fabric-prototype/blob/b9b264b32b6651ba4e15b96c236645e444e51f30/ethereum/migrations/2_deploy_contracts.js

**Comment by abcoathup** [comment]
Hi @phmatray 

I wondered why you needed `require('openzeppelin-test-helpers/configure')({web3})` in your migration script for the migration contract.  I assume you don't think you need it there.

I have it in the migration script for my ERC777 token as I need to deploy the ERC1820 registry when using a development (non-public) network.

```js
// File: 2_deploy.js
const Simple777Token = artifacts.require('Simple777Token');
const Simple777Recipient = artifacts.require('Simple777Recipient');

require('openzeppelin-test-helpers/configure')({ web3 });

const { singletons } = require('openzeppelin-test-helpers');

module.exports = async function (deployer, network, accounts) {
  if (network === 'development') {
    // In a test environment an ERC777 token requires deploying an ERC1820 registry
    await singletons.ERC1820Registry(accounts[0]);
  }

  await deployer.deploy(Simple777Token);
  const token = await Simple777Token.deployed();

  await deployer.deploy(Simple777Recipient, token.address);
};
```
My sample repo: https://github.com/abcoathup/Simple777Token

For support questions you can ask in the Community Forum: https://forum.openzeppelin.com/

**Comment by phmatray** [comment]
Indeed, it was to keep the example simple.
Just like you, it's as part of the development of an ERC777.

**Comment by abcoathup** [comment]
@phmatray thanks for explaining your example. 

Is there any other information that you can share about your environment e.g. versions, to help track it down? 

It would be great to be able to reproduce.

**Comment by frangio** [comment]
@phmatray This is interesting, I don't think we have a nice way of solving this at the moment.

We have some ideas on how to improve this that we'll be working on. For now I think the `try` block is ugly but it should work. Can you confirm that?

**Comment by phmatray** [comment]
@frangio: I confirm that it works, I was able to deploy my contracts on kovan

**Comment by nventuro** [comment]
Closed via #75
